rust/tw_evm/src/abi/prebuild/resource/erc4337.simple_account.sol

```diff
@@ -0,0 +1,139 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity ^0.8.12;
+
```

Comment:
is this file needed to be pushed?

Reply:
I think yes because I had to modify the original https://github.com/eth-infinitism/account-abstraction/blob/develop/contracts/samples/SimpleAccount.sol
So it will be easier to make any changes in the ERC4337 protocol later

Reply:
ok @rsrbk let's also review the change on this file please!

Reply:
Btw, the original `SimpleAccount.sol` has been updated 4 days ago!
https://github.com/eth-infinitism/account-abstraction/commit/0a9d55ca73cd309f4ec1cda28284d2d039e552b9
I can remove it now

Reply:
alright

Reply:
Done